Unikemet-17.0.0d3

eggrobin opened the pull request:
From @Ken-Whistler:

Michel has provided what he intends as the final version of Unikemet.txt for 17.0. I have pulled it into my framework and have done a quick check that the changes look reasonable.

This includes normative changes to kEH_Desc field values that the UTC will need to rubber-stamp this week, but also a bunch of changes to the FVal and Func fields which show up in the names list.

markusicu commented:
I am inclined to go with my PR now which includes the derived NamesList changes.

eggrobin commented:
OK, I will check that the files are the same since the diff was generated on top of this branch.

eggrobin commented:
```
git cherry-pick -n markus/uni17-ken-michel-july
git status
On branch unikenneth
Your branch is up to date with 'origin/unikenneth'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   unicodetools/data/ucd/dev/NamesList.txt
        modified:   unicodetools/data/ucd/dev/StandardizedVariants.txt
        modified:   unicodetools/data/ucd/dev/TangutSources.txt
```

So no difference on Unikemet.
